lib/perl/Genome/Model/SomaticValidation/Command/AlignReads.pm

```diff
@@ -105,5 +109,66 @@ sub execute {
     return 1;
 }
 
+sub _find_existing_alignments {
```

Comment:
I'd suggest naming this something like "_found_existing_alignments" or "_has_existing_alignments", since it's returning true/false, not the actual alignments it found.